core/schema/src/main/liquibase/19.0.2/changelog.xml

```diff
@@ -0,0 +1,145 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+  xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-2.0.xsd" >
+
+  <!-- NMS-9146: Assure that the Default location exists in the database -->
+  <changeSet author="cpape" id="19.0.2-assure-default-location-exists">
+    <preConditions onFail="MARK_RAN">
+      <sqlCheck expectedResult="0">
+        SELECT COUNT(id) FROM monitoringlocations WHERE id='Default';
+      </sqlCheck>
+    </preConditions>
+    <insert tableName="monitoringlocations">
+      <column name="id" value="Default"/>
+      <column name="monitoringarea" value="Default"/>
+    </insert>
+  </changeSet>
+
+  <!-- NMS-9146: Check for references to blank locations and associate them to the Default location -->
+  <changeSet author="cpape" id="19.0.2-fix-empty-locations-in-node-table">
+    <preConditions onFail="MARK_RAN">
+      <sqlCheck expectedResult="1">
+        select
+          case when exists (
+            select location from node where location=''
+          )
+          then 1
+          else 0
+        end;
```

Comment:
Sorry, you can only compare expectedResult to a single value and cannot create things like 'greater than' or 'lower than'. So, the recommended workaround is to use these 'case' statements.